bundles/org.openhab.binding.pilight/README.md

```diff
@@ -87,13 +88,13 @@ things from them.
 ### pilight.things
 
 ```java
-Bridge pilight:bridge:raspi "Pilight Daemon raspi" [ ipAddress="192.168.1.1", port=5000 ] {
+Bridge pilight:bridge:raspi "Pilight Daemon raspi" [ ipAddress="192.168.1.1", port=5000, backgroundDiscovery=false ] {
         Thing switch office "Office" [ name="office" ]
         Thing dimmer piano "Piano"  [ name="piano" ]
         Thing generic weather "Weather"  [ name="weather" ] {
             Channels:
-              State Number : temperature [ property="temperature"]
-              State Number : humidity [ property="humidity"]
+              Type number : temperature "Temperature" [ property="temperature"]
+              Type number : humidity "Humidity" [ property="humidity"]
```

Comment:
I have a doubt. Are you sure that number is ok and not Number ?

Reply:
If a use "Number" instead of "number" there is a parse error:

```
12:00:54.065 [WARN ] [el.core.internal.ModelRepositoryImpl] - Configuration model 'pilight.things' has errors, therefore ignoring it: [6,20]: mismatched input 'Number' expecting RULE_ID
[7,20]: mismatched input 'Number' expecting RULE_ID

```

Reply:
Ok